src/nodes.rs

```diff
@@ -251,6 +251,9 @@ pub struct NodeLink {
     /// Note this field is used for the `title` attribute by the HTML formatter even for images;
     /// `alt` text is supplied in the image inline text.
     pub title: String,
+
+    /// Whether this is a wikilink or not
+    pub wikilink: bool,
```

Comment:
>  it'd be nice if everyone didn't have to add it to their link definitions,

absolutely! I didn't realize we could that.

Reply:
uhh 

```
error: default values on `struct` fields aren't supported
   --> src/nodes.rs:256:23
    |
256 |     pub wikilink: bool = false,
    |                       ^^^^^^^^ help: remove this unsupported default value

```

Reply:
I think the only way to do this would be to move to the Builder pattern, which we use for like `RenderOptions`, but I think that's much bigger change I think.

Reply:
You know, the wiki code is pretty self-contained, so I could create `NodeWikilink`, rather than modifying the `NodeLink`. That should maintain backward compatibility.

Reply:
Hah, oops. I must've been thinking too much of Zig. `NodeWikilink` does sound like the way forward!

Reply:
`NodeWikiLink` implemented